earth_enterprise/src/server/wsgi/serve/publish/publish_manager_helper.py

```diff
@@ -1192,7 +1192,7 @@ def _GetPortableGlobesList(self, registered_portable_set):
     root = constants.CUTTER_GLOBES_PATH
     for name in os.listdir(root):
       # Ignore globes that are registered.
-      if name not in registered_portable_set:
+      if (name not in registered_portable_set) and (os.path.isfile(name)):
```

Comment:
Minor: Python usually uses less brackets.  E.g. `name not in registered_portable_set and os.path.isfile(name)`.
`